pymc3/math.py

```diff
@@ -22,3 +22,10 @@ def invlogit(x, eps=sys.float_info.epsilon):
 
 def logit(p):
     return tt.log(p / (1 - p))
+
+
+def flatten(tensors, outdim=1):
+    if not isinstance(tensors, list):
+        return _flatten(tensors, outdim)
```

Comment:
Hmm using a private method that's the same name as the other one. I'm not so sure about this.